terraform/daily_snapshot/prod/main.tf

```diff
@@ -36,7 +36,7 @@ module "daily_snapshot" {
   size            = "s-4vcpu-16gb-amd"      # droplet size
   slack_channel   = "#forest-notifications" # slack channel for notifications
   snapshot_bucket = "forest-archive"
-  forest_tag      = "2023-11-14-ddbdbb7"
+  forest_tag      = "latest"
```

Comment:
Let's use the fixed tag. It gives us more control and prevents potential issues that were present before when we had the latest tag.

Reply:
OK. `2023-11-19-911daab` should be sufficient then. Fixed